change/@fluentui-react-native-2020-07-07-19-11-38-lehon-contextmenu.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
I would say at the very least this is a patch. You've added to what the metapackage exports. I think in an ideal world this is a minor version bump.